### PR TITLE
Update dsl2_template branch after 1.13.1 release

### PR DESCRIPTION
**Pull request by drpatelh**
## PR checklist

 - [ ] This comment contains a description of changes (with reason)
 - [ ] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by KevinMenden — APPROVED** [review]
Alright I've been over it but didn't look at every single file in detail .... 
Looks good 👍 
Needs the new stuff in `dev` of course.

**Review by ggabernet — COMMENTED** [review]
In the `master` branch of tools the pipeline template is no longer under the {{cookiecutter.name_noslash}} directory. Should this be updated here as well?

**Comment by ggabernet** [comment]
> In the `master` branch of tools the pipeline template is no longer under the {{cookiecutter.name_noslash}} directory. Should this be updated here as well?

Ah sorry, I see now that it has been renamed, I said nothing! 😄

**Review by ggabernet — APPROVED** [review]
I've focused rather on the pipeline template code and it looks good to me too

File: .github/workflows/create-lint-wf.yml [hunk 1]
```diff
@@ -20,24 +22,23 @@ jobs:
 
       - name: Install Nextflow
         env:
-          CAPSULE_LOG: none
+            CAPSULE_LOG: none
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply .github/workflows/create-lint-wf.yml]: Yup. Fixed https://github.com/nf-core/tools/pull/927/commits/a42a0116c32e6a4a86f742c81a9b873ea90ca8e3

File: nf_core/lint/files_exist.py [hunk 2]
```diff
@@ -17,17 +16,36 @@ def files_exist(self):
...
+        assets/email_template.txt
+        assets/nf-core-PIPELINE_logo.png
+        assets/sendmail_template.txt
+        CHANGELOG.md
+        CODE_OF_CONDUCT.md
+        CODE_OF_CONDUCT.md
+        docs/images/nf-core-PIPELINE_logo.png
+        docs/output.md
+        docs/README.md
+        docs/README.md
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply nf_core/lint/files_exist.py]: Fixed https://github.com/nf-core/tools/pull/927/commits/4caa40e09c4a062cb321a530a94d0659443c269a

File: nf_core/lint/files_exist.py [hunk 3]
```diff
@@ -17,17 +16,36 @@ def files_exist(self):
...
+        .github/workflows/linting_comment.yml
+        .github/workflows/linting.yml
+        [LICENSE, LICENSE.md, LICENCE, LICENCE.md]  # NB: British / American spelling
+        assets/email_template.html
+        assets/email_template.txt
+        assets/nf-core-PIPELINE_logo.png
+        assets/sendmail_template.txt
+        CHANGELOG.md
+        CODE_OF_CONDUCT.md
+        CODE_OF_CONDUCT.md
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply nf_core/lint/files_exist.py]: Fixed https://github.com/nf-core/tools/pull/927/commits/4caa40e09c4a062cb321a530a94d0659443c269a

**Comment by drpatelh** [comment]
Ok. Everything is passing all conflicts have been resolved 🎉 Going to merge now and create a separate PR to `dev` for review!

**Pull request merged**